### ISSUE TITLE
is any way to use a pork pie for gathering yield or for fishing speed food?, let me know pls thanks for your work! 

### ISSUE THREAD

**Issue by TKRAIZER1** [CapochDev/FishbotKEKW#13]

**Comment by Thoks10a** [comment]
I didn't understand your question, do you want to put a new function to consume food also besides bait?

**Comment by TKRAIZER1** [comment]
> I didn't understand your question, do you want to put a new function to consume food also besides bait?

no, the fish consume always need to be thats 1st slot, the slot for potions is empty and do have an use for that, i was think to add consume pork pie in potions sloth for yield and gathering % to get more fish. i dont know how to do, if you can help me please, (pic from google 1slots its for fish bait)
![image](https://github.com/user-attachments/assets/525857a9-4462-4f59-8e66-608e273abe12)
